src/vmm/Cargo.toml

```diff
@@ -56,7 +59,9 @@ itertools = "0.13.0"
 proptest = { version = "1.5.0", default-features = false, features = ["std"] }
 
 [features]
+default = []
 tracing = ["log-instrument"]
+debug = ["arrayvec", "gdbstub", "gdbstub_arch"]
```

Comment:
Maybe name it `gdb`. `debug` can mean many things.

Reply:
Good point updated